### DISCUSSION TITLE
EPSON L395

### DISCUSSION THREAD

**Discussion by antonio-queiros** [Ircama/epson_print_conf#1] [Ideas]
Hi! I'm looking for a Waste Reset for L395. What I understood that this printer is not implemented yet. I have one here. Can we try to implement it together?

**Comment by Ircama** [comment]
Unfortunately, I do not know that printer. You might try the [Epson Maintenance Reset Utility](https://epson.com/epsonstorefront/orbeon/fr/us_regular_s03/us_ServiceInk_Pad_Reset/new) at your risk. I think that tool requires the USB connection (and does not support WiFi). If you succeed, you could trace the USB traffic via Wireshark and post the log.

**Comment by Ircama** [comment]
L395 added
